index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script
+      src="https://kit.fontawesome.com/9f61ce6d5c.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="css/fontawesome/css/font-awesome.min.css" />
+    <link
+      rel="stylesheet"
+      href="https://bootswatch.com/4/yeti/bootstrap.min.css"
+    />
+    <title>MyLibrary App</title>
+  </head>
+  <body>
+    <div class="container mt-4">
+      <h1 class="display-4 text-center">
+        <i class="fas fa-book-open text-primary"></i> My<span
+          class="text-primary"
+          >Library</span
+        >Store
+      </h1>
+      <form id="book-form">
+        <div class="form-group">
+          <label for="title">Title</label>
+          <input type="text" id="title" class="form-control" />
+        </div>
+        <div class="form-group">
+          <label for="author">Author</label>
+          <input type="text" id="author" class="form-control" />
+        </div>
+        <div class="form-group">
+          <label for="pages">Number of pages</label>
+          <input type="text" id="page" class="form-control" />
```

Comment:
Opps, you set the input page number to text, kindly validate this, users should not be able to enter non-digit data.
![image](https://user-images.githubusercontent.com/31897434/87005702-5526bf00-c1b7-11ea-9e16-cf34f178bf75.png)